test/e2e/externalservices_kubernetes_test.go

```diff
@@ -105,6 +105,7 @@ metadata:
 
 	var cluster Cluster
 	var clientPod *v1.Pod
+	var deployOptsFuncs []DeployOptionsFunc
```

Comment:
are changes in this file required or can we revert it?

Reply:
Let's keep these for uniformity of tests